CMakeLists.txt

```diff
@@ -8,4 +8,6 @@ include(projectheader)
 project_header(modbus-cpp)
 
 add_subdirectory(src)
+add_subdirectory(tests)
+# add_subdirectory(tools)
```

Comment:
Que? :D

Reply:
Already preparing for implementation of test-tools. Interactive configuration and communication